UnityProject/Assets/Scripts/Objects/Wallmounts/ClockWallmount.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using AddressableReferences;
+using Managers;
+using Messages.Server.SoundMessages;
+using Mirror;
+using Systems.Explosions;
+using UI.Systems.Tooltips.HoverTooltips;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace Objects.Wallmounts
+{
+	public class ClockWallmount : NetworkBehaviour, IExaminable, IHoverTooltip, IEmpAble, ICheckedInteractable<HandApply>
+	{
+		[SyncVar] private DateTime UST;
+
+		[SerializeField] private AddressableAudioSource tickSound;
+
+		private bool messedWith = false;
+		private const float TICK_TIME = 1.75f;
```

Comment:
is this because the sound you are playing has two ticks?

Reply:
Yup